### ISSUE TITLE
Adding templates/customers/account.liquid error

### ISSUE THREAD

**Issue by cleo06** [Shopify/themekit#516]
I get an error when I try to add customers/account.liquid to the templates folder

`16:37:47 [development] Received Update event on templates/customers
[development] Asset Perform Update to templates/customers at host georgie-porgie-test.myshopify.com
	Status: 422 Unprocessable Entity
	Errors: Must have a .liquid file extension`

Fixed by increasing timeout in config

**Comment by Jrschellenberg** [comment]
Hi, I was having a similar error. I am using 

MAC High Sierra 10.13.4
Themekit version: Newest.
IDE: Webstorm, 2018.1.4

I Believe I had solved this issue, it is not necessarily themekits issue, but rather the IDE that I was using.

I solved the issue by disabling safe write. Instructions can be found [here](https://stackoverflow.com/questions/23864827/jb-bak-and-jb-old-files-in-pycharm)

Hope this helps your issue.
